src/js/view/controls/controls.js

```diff
@@ -117,14 +108,28 @@ export default class Controls {
         this.div.appendChild(controlbar.element());
 
         // Settings Menu
+        let lastState = null;
         const visibilityChangeHandler = (visible) => {
+            const state = model.get('state');
+            const settingsInteraction = { reason: 'settingsInteraction' };
+
             utils.toggleClass(this.div, 'jw-settings-open', visible);
+            if (getBreakpoint(model.get('containerWidth')) < 2) {
+                if (state === STATE_PLAYING) {
```

Comment:
Though this seems to work, I'd rather you be explicit by incorporating `visible` so we're calling play/pause based on whether we're opening or closing the settings menu.